core/templates/dev/head/pages/exploration_player/PlayerServices.js

```diff
@@ -320,13 +320,20 @@ oppia.factory('ExplorationPlayerService', [
             interactionRulesService));
 
         if (!_editorPreviewMode) {
+          // Replace this with the isClassifiedExplicitlyOrGoesToNewState method
```

Comment:
Why not just do this now?

Reply:
@seanlip This function is being implemented in your current PR :)

Reply:
Ah sorry, I didn't realize it's still open. You can merge and add it now :)